__mocks__/got.js

```diff
@@ -1,58 +1,10 @@
 const fs = require('fs');
+const gotSwapi = require('./gotSwapi.js');
```

Comment:
This is not what I meant when we discussed this.

In your other PR, you've created a "Swapi" class for querying the Star Wars API. That is module you need to mock, once it gets merged, not doing this